prime-dotnet-webapi/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+services:
+######################################################### Sign-in #
+  dotnet-webapi:
+    container_name: primeapi
+    restart: always
+    build:
+      context: .
+    environment:
+      DB_CONNECTION_STRING: "host=postgres;port=5432;database=postgres;username=postgres;password=postgres"      
+      JWT_SIGNING_KEY: "SdqlA0Y5e6HMAK1wm6iHHHz2u37XGtlo"      
+      ASPNETCORE_HTTPS_PORT: "5001"
+      ASPNETCORE_ENVIRONMENT: "Development"
+    ports:
+      - "5000:80"
```

Comment:
Will this need to map 5000 to port 8080 now, not port 80?

Reply:
We cannot map to port 80 locally on a container.  We can redirect the public port 80 to a container on 8080 or some other high-range port but not directly to anything low-range (like 1024 or lower).  It's a bad practice from a network perspective.

Reply:
Yes, so we probably need to have the container running on port 8080, not port 80 - since OpenShift won't allow port 80.